core/templates/dev/head/css/oppia.css

```diff
@@ -870,6 +870,39 @@ textarea {
   border: none;
 }
 
+/*Prevent Exploration Player From collapse On Small Screen*/
+@media(max-width:768px) {
+/* Disabling navbar-collapse */
```

Comment:
Sorry, i forgot to remove this comment. In my opinion we can actually remove navbar-collapse while addressing #2741 .